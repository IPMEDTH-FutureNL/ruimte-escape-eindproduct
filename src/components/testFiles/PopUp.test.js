import renderer from 'react-test-renderer';
import Popup from "../PopUp.js";

it('changes the pop-up text when needed', () => {
    const component = renderer.create(
        <Popup text="Test of input text wordt veranderd"></Popup>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})