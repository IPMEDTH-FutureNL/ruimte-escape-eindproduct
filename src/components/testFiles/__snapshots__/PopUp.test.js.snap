// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the pop-up text when needed 1`] = `
<div
  className="pop-up"
  id="popup"
>
  <p
    className="pop-up-text"
  >
     
    Test of input text wordt veranderd
     
  </p>
  <button
    className="button primary"
    onClick={[Function]}
  >
    Verder!
  </button>
</div>
`;
